backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/openitems_fibu/report_TabularView.jrxml

```diff
@@ -49,14 +48,11 @@ FROM
 		i.C_BPartner_ID,
 		i.IsSOTrx,
 		i.DateInvoiced,
-		EXISTS (SELECT 0 FROM C_PaySelectionLine psl WHERE psl.processed = 'Y' AND psl.C_Invoice_ID = i.C_Invoice_ID AND psl.isActive = 'Y') AS PassForPayment,
+		EXISTS (SELECT 0 FROM C_PaySelectionLine psl WHERE psl.C_Invoice_ID = i.C_Invoice_ID AND psl.isActive = 'Y') AS PassForPayment,
```

Comment:
why removing `psl.processed = 'Y'`?

Reply:
Because the column does not exist anymore. Apparently this jasper was broken from sometime.